app/scripts/controllers/preferences.test.js

```diff
@@ -109,6 +113,24 @@ describe('preferences controller', () => {
     });
   });
 
+  describe('onAccountRemoved', () => {
+    it('should call removeAddress method with the correct address when the `accountRemoved` event is emitted.', () => {
+      const mockAddress = '0x1234567890abcdef';
+      const removeAddressMock = jest.fn();
+      preferencesController.removeAddress = removeAddressMock;
+      mockEventEmitter.emit('accountRemoved', mockAddress);
+      expect(removeAddressMock).toHaveBeenCalledWith(mockAddress);
```

Comment:
Perhaps here we should check that whenever `onAccountRemoved` is called the passed address is actually removed from the `PreferencesController` state

Reply:
@mikesposito just below this test we have a test suit for removeAddress that's why I don't wanted to retest it